modules/gdnative/godot/godot_plane.cpp

```diff
@@ -28,48 +28,149 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 #include "godot_plane.h"
+#include "core/variant.h"
 
-#include "math/plane.h"
+#include "core/math/plane.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-void _plane_api_anchor() {
+void _plane_api_anchor() {}
+
+void GDAPI godot_plane_new_with_reals(godot_plane *r_dest, const godot_real p_a, const godot_real p_b, const godot_real p_c, const godot_real p_d) {
+
+	Plane *dest = (Plane *)r_dest;
+	memnew_placement(dest, Plane(p_a, p_b, p_c, p_d));
```

Comment:
Again, `Plane` doesn't need a non local lifetime, just doing `*dest = Plane(p_a, p_b, p_c, p_d);` is enough.